gusto/core/io.py

```diff
@@ -21,7 +21,7 @@ class GustoIOError(IOError):
     pass
 
 
-def pick_up_mesh(output, mesh_name):
+def pick_up_mesh(output, mesh_name, comm=COMM_WORLD):
```

Comment:
We had a brief chat about the addition of the `comm` argument here last week -- this makes sense to me now.